app/models/user.rb

```diff
@@ -2,4 +2,14 @@ class User < ActiveRecord::Base
   include Clearance::User
 
   has_many :itineraries
+
+  def testing_hound
+
+    true ? 'hi' : "bye"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.